stable/jenkins/values.yaml

```diff
@@ -6,7 +6,7 @@
 Master:
   Name: jenkins-master
   Image: "gcr.io/kubernetes-charts-ci/jenkins-master-k8s"
-  ImageTag: "v0.6.0"
+  ImageTag: "v0.7.0"
```

Comment:
@seanknox, the rest of the changes don't make sense without having the updated docker image for the chart to pull down, IMO.